README.md

```diff
@@ -1,70 +1,13 @@
-# Getting Started with Create React App
+### Summary
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a simple react todo-list saved data in local storage.
 
-## Available Scripts
+### The problem solved in this app
 
-In the project directory, you can run:
+When refreshing the browser, the status of the todo-list is kept in local storage.
```

Comment:
The todo list is not solving the problem. This should be in the PR description but not in the readme